README.md

```diff
@@ -1,17 +1,17 @@
 # mpl-interactions
-[![Documentation Status](https://readthedocs.org/projects/mpl-interactions/badge/?version=latest)](https://mpl-interactions.readthedocs.io/en/latest/?badge=latest)[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/ianhi/mpl-interactions/master?urlpath=lab) (warning: this will be laggy)
+[![Documentation Status](https://readthedocs.org/projects/mpl-interactions/badge/?version=latest)](https://mpl-interactions.readthedocs.io/en/latest/?badge=latest)[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/ianhi/mpl-interactions/master?urlpath=lab) (Warning: binder launch will be laggy)
```

Comment:
```suggestion
[![Documentation Status](https://readthedocs.org/projects/mpl-interactions/badge/?version=latest)](https://mpl-interactions.readthedocs.io/en/latest/?badge=latest)[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/ianhi/mpl-interactions/master?urlpath=lab) 

(Warning: The interactions will be laggy when on binder )
```


Also, if you haven't already and are curious to try the library yourself then binder is a good way to do that.
1. Once it launches use the filebrowser on the left to bavigate to the examples directory
2. I'd recommend `interactive_plot-examples` - double click to open
3. click in the first code block and press `shift-enter` to start running the contents of cells.